### DISCUSSION TITLE
I need help to solve the  "SyntaxError: Unexpected token '?'"

### DISCUSSION THREAD

**Discussion by MinLee0210** [vercel/community#7350] [Help]
### Summary

I need help to solve the  "SyntaxError: Unexpected token '?'"

### Steps to Reproduce

I am learning Flask and trying to deploy my first app with vercel. However, whenever I type `vercel`, my terminal returns "SyntaxError: Unexpected token '?'". How can I overcome this? 

I install via the command line `npm i -g vercel`. I am using `npm@8.5.1`. 

The problem is caughted on `./vercel/dist/index.js:120`

![Screenshot from 2024-07-02 20-47-34](https://github.com/vercel/community/assets/57653278/aceaab8c-af9c-4da3-852f-2ee6f239a643)

**Comment by pawlean** [comment]
Hi @MinLee0210!

Congrats on starting the journey in learning Flask 🥳 

Have you checked if you're running a compatible version of Node.js? Vercel supports Node.js 14 and above. You can check your Node.js version by running node -v in your terminal.

[a comment by an automated account is not shown]
